pkg/rtc/subscriptionmanager.go

```diff
@@ -36,9 +36,9 @@ var (
 	reconcileInterval = 3 * time.Second
 	// amount of time to give up if a track or publisher isn't found
 	// ensuring this is longer than iceFailedTimeout so we are certain the participant won't return
-	notFoundTimeout = iceFailedTimeout
+	notFoundTimeout = time.Minute
```

Comment:
the bar for notFound should be very high, since it would reset desired state for the subscription.

In case of temporary resolver issues, this would cause tracks to be unsubscribed unintentionally.